CMakeLists.txt

```diff
@@ -106,6 +106,7 @@ option(USE_MGBA "Enables GBA controllers emulation using libmgba" ON)
 option(ENABLE_AUTOUPDATE "Enables support for automatic updates" ON)
 option(STEAM "Creates a build for Steam" OFF)
 option(USE_RETRO_ACHIEVEMENTS "Enables integration with retroachievements.org" ON)
+option(RC_CLIENT_SUPPORTS_HASH "Enables retroachievements client hashing" ON)
```

Comment:
What exactly does this option do, anyway? Is there a need for this to be separately toggleable?

Reply:
I believe it was Stenzek who requested a change to rcheevos to disable its hashing functionality by default for PCSX2. As this function is now off by default, I need to enable it in order to use it.

Reply:
What I mean is, why does it need to be toggleable by someone building Dolphin? Can't we just always enable it since Dolphin will (should) always hash?

Reply:
I would certainly like to, but I was having trouble getting it to do that when building. What would you recommend?

Reply:
Oh, fair. The obvious solution I didn't think of. Thanks!